### DISCUSSION TITLE
[Projects Beta] Create draft issue template link

### DISCUSSION THREAD

**Discussion by jessehouwing** [community/community#15611] [Projects and Issues]
When opening a new issue, I can send people a link like:

```
https://github.com/xxx/Courseware/issues/new?title=&labels=MyLabel
```

It would be handy when we can do the same for draft issues:

```
https://github.com/orgs/xxx/projects/5/views/1/new?title=&labels=bug&column=To%20Do
```

Where the draft issue would be created in the `To Do` column and have the correct labels and assignee etc.

**Comment by PixelyIon** [comment]
This would be extremely useful, currently there's no way to share filled out draft items which would help out a lot with coordination in a team.

**Comment by mffap** [comment]
Agreed this would be very useful. Would love to see also to be able to convert a draft based on an issue template, seems related to me.

**Comment by mdrabic** [comment]
This is similar to https://github.com/github-community/community/discussions/16610 and these could probably be merged together. Our team would love to have this functionality.
